run_scripts/run_pipeline.sh

```diff
@@ -28,10 +28,3 @@ DATA_BACKUPS_DIR=$8
 
 ./4_coda_add.sh "$CODA_PUSH_CREDENTIALS_PATH" "$CODA_TOOLS_ROOT" "$DATA_ROOT"
 
```

Comment:
I think you want to run the backup script from here?

Reply:
Yes Updated PTAL!